sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -518,7 +516,7 @@ def __init__(
     @classmethod
     def _from_generated(cls, mark, page):
         return cls(
-            confidence=mark.confidence,
+            confidence=adjust_confidence(mark.confidence),
```

Comment:
ohh in which cases is confidence not returned?

Reply:
I added this since we call this method on all service returned confidences and it didn't make sense to leave this one out

Reply:
fair. I haven't seen it for SelectionMarks so I was worried I had to make the change or there was a case I needed to add tests for.